### PR TITLE
Error message is not appearing in web browser

### LINKED ISSUE #91: Bug: Error message is not appearing in web browser
**Describe the bug**
Error message is not appearing in web browser, when asked for example: for data that not exists.

**To Reproduce**
1. Prepare Web Frontend to get some data, that doesn't exists,
2. Go to Web Frontend via web browser,
3. Open dev tools,
4. Move to console tab,

To make sure, that Web Backend is working properly, follow this path:

1. Go to API debugging software (like Postman, etc.),
2. Send request to Web Backend, asking for value, that does not exists,
3. See results,

**Expected behavior**
Error object returned from Web Backend is provided in console on web browser.

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**
Web Frontend
Web Backend

### PR DESCRIPTION
**Pull request by Lukash88**
## Summary

Error messages are appearing in web browser (DevTools)

## Details

- Changes in  `ToDoTasks`  `Repository` and `Service`,
- Error handling middleware refactored - swap  switch  expression to switch statement ,
 and assigned  http response code to each response case.

**Screenshots:**

image

## References

No external references needed.

### CODE REVIEW

File: binder-web-backend/Binder.Application/Services/Middleware/ErrorHandlingMiddleware.cs [hunk 1]
```diff
@@ -25,27 +25,38 @@ public async Task InvokeAsync(HttpContext context)
             }
             catch (Exception e)
             {
-                ProblemDetails problemDetails = e switch
+                ProblemDetails problemDetails;
```
Comment (StraykerPL) [inline binder-web-backend/Binder.Application/Services/Middleware/ErrorHandlingMiddleware.cs]: Add null to declaration.

File: binder-web-backend/Binder.Application/Services/Middleware/ErrorHandlingMiddleware.cs [hunk 2]
```diff
@@ -25,27 +25,38 @@ public async Task InvokeAsync(HttpContext context)
...
-                        Detail = notFoundException.Message
-                    },
-                    _ => new ProblemDetails
-                    {
-                        Title = ExceptionConstants.UnexpectedErrorMessage,
-                        Status = (int)HttpStatusCode.InternalServerError,
-                        Detail = e.Message
-                    },
-                };
+                    case InvalidOperationException invalidOperationException:
```
Comment (StraykerPL) [inline binder-web-backend/Binder.Application/Services/Middleware/ErrorHandlingMiddleware.cs]: There's no need to make case variable - use `e.Message`.

File: binder-web-backend/Binder.Application/Services/ToDoTasksService.cs [hunk 3]
```diff
@@ -16,8 +17,7 @@ public ToDoTasksService(IToDoTasksRepository repository)
 
         public ICollection<ToDoTask> GetTasksForTable(int tableId)
         {
-            return _repository.GetTasksByTable(tableId) ??
-                throw new ArgumentException(ExceptionConstants.ResourceNotFoundMessage);
+            return _repository.GetTasksByTable(tableId) ?? throw new NotFoundException(ExceptionConstants.ResourceNotFoundMessage);
```
Comment (StraykerPL) [inline binder-web-backend/Binder.Application/Services/ToDoTasksService.cs]: Why returned to old styling?

**Review by StraykerPL — CHANGES_REQUESTED** [review]

**Comment by StraykerPL** [comment]
Please remember next time about commit name styling.

**Comment by StraykerPL** [comment]
Please provide example screenshot to description.

File: binder-web-backend/Binder.Application/Services/Middleware/ErrorHandlingMiddleware.cs (hunk 1 above)
Comment (Lukash88) [reply binder-web-backend/Binder.Application/Services/Middleware/ErrorHandlingMiddleware.cs]: Resolved 60b5acc.

File: binder-web-backend/Binder.Application/Services/ToDoTasksService.cs (hunk 3 above)
Comment (Lukash88) [reply binder-web-backend/Binder.Application/Services/ToDoTasksService.cs]: Back to  previous style.

Resolved 7f4c386.

File: binder-web-backend/Binder.Application/Services/Middleware/ErrorHandlingMiddleware.cs (hunk 2 above)
Comment (Lukash88) [reply binder-web-backend/Binder.Application/Services/Middleware/ErrorHandlingMiddleware.cs]: Resolved 60b5acc.

**Review by StraykerPL — APPROVED** [review]

**Pull request merged**